app/views/layouts/_navbar.html.erb

```diff
@@ -1,10 +1,17 @@
-<div class="flex bg-black h-24 md:h-14 px-6 gap-10 justify-around m-auto" >
-    <%= link_to "index", class: 'h-10 w-48 my-auto' do %>
+<div class="flex bg-black px-6 gap-10 justify-around m-auto lg:h-24 h-14 sticky items-center" >
+    <%= link_to root_path, class: 'my-auto' do %>
       <%= image_tag("logo") %>
     <% end %>
-    <%= link_to "shopping_cart_path", class: 'h-10 w-18 p-2 border inline-flex text-white rounded my-auto' do %>
-      <span class="font-sans text-sm text-white hidden md:inline">Shopping Cart</span>
-      <%= image_tag("cart") %>
+    <div class="flex flex-col md:flex-row items-start md:items-center justify-betwee w-full">
+    <%= form_with url: products_path, method: :get, data: { controller: 'search-product', search_product_target: 'form' } do |f| %>
```

Comment:
not for now, but I think it's a bit strange that generic code that lives in the layout, always hit a very specific route like `products_path`